src/index.js

```diff
@@ -1,34 +1,186 @@
 import './style.css';
-
-/* eslint-disable  no-unused-vars */
-import refresh from './icons/refresh.svg';
-
 import enter from './icons/enter.svg';
+import refresh from './icons/refresh.svg';
+import ToDoList from './modules/List.js';
+import Task from './modules/Task.js';
+import Actions from './modules/actions.js';
```

Comment:
- [Optional] To get better at such an exercise, you should consider Implementing the `drag / drop and sorting` functionality in a new file and add it as a module. This will enable users to rearrange their tasks according to importance or other factors they believe matter.